admin/spec/lib/validator/pupil-validator.spec.js

```diff
@@ -424,6 +424,13 @@ describe('pupil validator', function () {
         expect(validationError.get('gender')).toBe('Gender must be M or F')
         done()
       })
+      it('can be accepted in lowercase', async (done) => {
```

Comment:
do we have a test for upperCase too?

Reply:
the existing successful scenario includes a gender in uppercase.(getBody method)